201-data-factory-ftp-hive-blob/azuredeploy.parameters.json

```diff
@@ -3,13 +3,13 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "ftpHost": {
-      "value": "13.93.203.101"
+      "value": "FTP server name or IP address"
```

Comment:
Just remove these from the file altogether - otherwise people tend to deploy with the value unchanged.